src/ClientRuntime/Microsoft.Rest.ClientRuntime.E2E.Tests/ScenarioTests/VMTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace Microsoft.Rest.ClientRuntime.E2E.Tests.ScenarioTests
+{
+    using Microsoft.Azure.Management.Compute;
+    using Microsoft.Azure.Management.Compute.Models;
+    using Microsoft.Azure.Management.Resources;
+    using Microsoft.Azure.Management.Resources.Models;
+    using Microsoft.Azure.Management.Storage.Models;
+    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+    using Microsoft.Rest.ClientRuntime.E2E.Tests.TestAssets;
+    using Rest.Azure;
+    using System.Net;
+    using Xunit;
+    using System.Threading.Tasks;
+    using System.Diagnostics;
+    using System;
+    using System.Net.Http;
+
+    public class VMTests : E2ETestBase
+    {
+        const string prefix = "e2etests";
+        const string resourcePrefix = "res";
+        const string storagePrefix = "sto";
+        const string networkPrefix = "net";
+        const string vmPrefix = "vm";
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
```

Comment:
@hovsepm I am used to StyleCop, so I keep adding xml comments for public functions.